File: python/bee1061.py
dia_inicio = int(input().split()[1])
hi, mi, si = map(int, input().split(' : '))
dia_final = int(input().split()[1])
hf, mf, sf = map(int, input().split(' : '))

hi = hi*3600
hf = hf*3600
mi = mi*60
mf = mf*60

dia_segundos = (dia_final - dia_inicio)*86400

inicio_segundos = hi + mi + si
final_segundos = hf + mf + sf

total_segundos = (dia_segundos - inicio_segundos) + final_segundos

dias = total_segundos // 86400
r = total_segundos % 86400
horas = r // 3600
r = r % 3600
minutos = r // 60
r = r % 60
segundos = r

print(f'{dias} dia(s)')
print(f'{horas} hora(s)')
print(f'{minutos} minuto(s)')
print(f'{segundos} segundo(s)')

File: java/bee1061.java
import java.util.Scanner;

public class bee1061 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        input.next();
        int diaInicio = input.nextInt();
        
        input.nextLine();
        String horarioInicioStr = input.nextLine().trim();
        String[] partesInicio = horarioInicioStr.split(":");

        int horaInicio = Integer.parseInt(partesInicio[0].trim());
        int minutoInicio = Integer.parseInt(partesInicio[1].trim());
        int segundoInicio = Integer.parseInt(partesInicio[2].trim());

        input.next();
        int diaFim = input.nextInt();
        
        input.nextLine();
        String horarioFimStr = input.nextLine().trim();
        String[] partesFim = horarioFimStr.split(":");

        int horaFim = Integer.parseInt(partesFim[0].trim());
        int minutoFim = Integer.parseInt(partesFim[1].trim());
        int segundoFim = Integer.parseInt(partesFim[2].trim());

        int tempoInicioSegundos = calculaTempoSegundos(diaInicio, horaInicio, minutoInicio, segundoInicio);
        int tempoFimSegundos = calculaTempoSegundos(diaFim, horaFim, minutoFim, segundoFim);

        int diferencaSegundo = tempoFimSegundos - tempoInicioSegundos;

        Tempo tempoFormatado = new Tempo(diferencaSegundo);

        tempoFormatado.mostraTempo();

        input.close();
    }

    public static int calculaTempoSegundos(int dia, int hora, int minuto, int segundo) { 
        return dia*86400 + hora*3600 + minuto*60 + segundo;
    }
 
}

class Tempo {
    private int tempo;

    Tempo(int tempo) {
        this.tempo = tempo;
    }

    public void mostraTempo() {
        int dias = tempo / 86400;
        int resto = tempo % 86400;

        int horas = resto / 3600;
        resto %= 3600;

        int minutos = resto / 60;
        int segundos = resto % 60;


        System.out.println(dias + " dia(s)");
        System.out.println(horas + " hora(s)");
        System.out.println(minutos + " minuto(s)");
        System.out.println(segundos + " segundo(s)");
    }
}